modules/liquidity-pools/src/lib.rs

```diff
@@ -44,6 +48,7 @@ decl_storage! {
 		pub Owners get(fn owners): map hasher(blake2_256) T::LiquidityPoolId => Option<T::AccountId>;
 		pub LiquidityPoolOptions get(fn liquidity_pool_options): double_map hasher(blake2_256) T::LiquidityPoolId, hasher(blake2_256) T::CurrencyId => Option<LiquidityPoolOption>;
 		pub Balances get(fn balances): map hasher(blake2_256) T::LiquidityPoolId => T::Balance;
+		pub MinAdditionalCollateralRatio get(fn min_additional_collateral_ratio): Option<Permill>;
```

Comment:
No need to make this `Option`, make it `Permill` and should simplify a lot of code.
It will default to zero which is fine.

Reply:
if `min_additional_collateral_ratio()` return Permil(0),  shuold enforce  call `set_min_additional_collateral_ratio`?

Reply:
what do you mean? enforce what?
set_min_additional_collateral_ratio can set it to any value, including 0

Reply:
1. If not set min_additional_collateral_ratio, so `min_additional_collateral_ratio()`return default value `Permil(0)`. 
2.If lp owner not set additional_collateral_ratio, so `get_additional_collateral_ratio()` will 
return the `min_additional_collateral_ratio()`, is `Permil(0)`.
In this case, no people to set `min_additional_collateral_ratio` or `additional_collateral_ratio`, it will run with `Permil(0)`.
Is this what is expected？If yes, I'll remove `Error::NoAdditionalCollateralRatio`

Reply:
I see. In that case you should add `config()` so we can set the value in chain spec.